api/sp.go

```diff
@@ -49,13 +50,43 @@ func (w Wrapper) UpdateServiceProvider(ctx echo.Context) error {
 
 func (w Wrapper) RegisterEndpoint(ctx echo.Context) error {
 	ep := domain.EndpointProperties{}
+
 	if err := ctx.Bind(&ep); err != nil {
 		return echo.NewHTTPError(http.StatusInternalServerError, err.Error())
 	}
-	err := w.SPService.RegisterEndpoint(ep)
-	if err != nil {
+
+	if err := w.SPService.RegisterEndpoint(ep); err != nil {
 		return echo.NewHTTPError(http.StatusInternalServerError, err.Error())
 	}
+
+	// Automatically set NutsComm endpoints for customers
```

Comment:
I also wrote the same code in `api/sp.go:33`, because `NutsComm` can be registered by manually adding the endpoint (which I didn't intend) and using the input field. Maybe we can at least deduplicate the code?